_commands/basics/seq.md

```diff
@@ -0,0 +1,76 @@
+---
+---
+
+seq
+-------
+### Overview
+
+`seq`prints a sequence of numbers to standard ouput.
+
+~~~ bash
+$ seq [options]... [*FIRST* [*STEP*]] *LAST*... 
+~~~
+
+<!--more-->
+
+where `seq` prints numbers from *FIRST* to *LAST* by an increment *STEP*.
+Default: *FIRST* & *STEP* are both 1. Each number is printed on its own line. Seq can be used on all real numbers.
+
+### Useful Options / Examples
+
+#### `-f, --format=`*FORMAT*`
```

Comment:
This is rendering with an extra backtick 
<img width="297" alt="screen shot 2016-03-25 at 11 06 35 pm" src="https://cloud.githubusercontent.com/assets/5882053/14057685/74e17e2c-f2de-11e5-825c-bb8dcc1f4c52.png">